lesson6/src/async-await.js

```diff
@@ -0,0 +1,17 @@
+async function getData(url) {
+    try {
+        const response = await fetch(url);
+        const data = await response.json();
+        console.log('Отримані дані:', data);
+        const summary = processData(data);
+        console.log('Результат обробки:', summary);
+    } catch (error) {
+        console.error('Помилка:', error);
+    }
+}
+
+function processData(data) {
+    return `Кількість елементів: ${data.length}`;
+}
+
+getData('https://jsonplaceholder.typicode.com/posts');
```

Comment:
як ми викликаємо асинхронну функцію?

```suggestion
await getData('https://jsonplaceholder.typicode.com/posts');
```